source/components/JsxParser.js

```diff
@@ -32,6 +32,7 @@ export default class JsxParser extends Component {
   }
 
   handleNewProps = (props) => {
+    this.bindings = (props.bindings || {})
```

Comment:
This indirection seems unnecessary. `props` is already available, and `this.bindings` is only used below.